superset/assets/package.json

```diff
@@ -121,7 +123,9 @@
     "supercluster": "https://github.com/georgeke/supercluster/tarball/ac3492737e7ce98e07af679623aad452373bbc40",
     "underscore": "^1.8.3",
     "urijs": "^1.18.10",
-    "viewport-mercator-project": "^5.0.0"
+    "url-polyfill": "^1.0.13",
+    "viewport-mercator-project": "^5.0.0",
+    "whatwg-fetch": "^2.0.4"
```

Comment:
Would ❤️ approval from @williaster and/or @graceguo-supercat on this import as a preferred method to eventually replace jQuery in the codebase

Reply:
What I meant was not jquery vs fetch, but npm's `fetch` package vs `whatwg-fetch` which are different things. I have no opposition with either but should make a conscious decision.